topi/python/topi/x86/conv2d_transpose.py

```diff
@@ -0,0 +1,101 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+# pylint: disable=invalid-name,unused-variable,unused-argument,no-member
+"""Conv2D Transpose schedule on x86"""
+
+import tvm
+from tvm import autotvm
+from .. import generic, tag
+from ..nn.conv2d_transpose import conv2d_transpose_nchw
+from ..nn.dilate import dilate
+from ..nn.pad import pad
+from ..nn.util import get_pad_tuple
+from ..util import simplify
+
+@autotvm.register_topi_compute(conv2d_transpose_nchw, 'cpu', ['direct'])
+def _declaration_conv2d_transpose(cfg, data, kernel, strides, padding, out_dtype):
+    return _declaration_conv2d_transpose_impl(cfg, data, kernel, strides, padding, out_dtype)
+
+def _declaration_conv2d_transpose_impl(cfg, data, kernel, strides, padding, out_dtype):
```

Comment:
can we put this function to `nn/conv2d_transpose.py` and have `nn.conv2d_transpose_nchw` and `_declaration_conv2d_transpose` share the implement?

Reply:
Makes sense. But I am assuming that `cfg` will be used in the future within this function. If we move it to `nn/conv2d_transpose.py`, I am not sure how to deal with it. What do you suggest? Thanks!

Reply:
Anyway, I modified as you suggested and put a `TODO` for now.